pkg/bgpv1/gobgp/state.go

```diff
@@ -92,6 +92,11 @@ func (g *GoBGPServer) GetPeerState(ctx context.Context) (types.GetPeerStateRespo
 			}
 		}
 
+		if peer.GracefulRestart != nil {
+			peerState.GracefulRestart = peer.GracefulRestart.Enabled
+			peerState.GracefulRestartRestartTimeSeconds = int64(peer.GracefulRestart.RestartTime)
```

Comment:
hmm, gobgp does not set PeerRestartTime [here](https://github.com/osrg/gobgp/blob/master/internal/pkg/config/util.go#L550-L558) 

I will create upstream PR to fix it. In the meantime, we will just not display peer restart time - since it will be 0

Reply:
https://github.com/osrg/gobgp/pull/2660